Content/StructureResolver.php

```diff
@@ -167,10 +178,15 @@ private function getStructureData(StructureInterface $structure): array
             $created = $document->getCreated();
         }
 
-        $type = 'page';
+        $type = 'unknown';
         if (method_exists($structure, 'getContent') && method_exists($structure->getContent(), 'getTemplateType')) {
             // used for content-bundle to determine current template type
```

Comment:
can we change this to `determine type for structure that is implemented based on the SuluContentBundle`?